params.py

```diff
@@ -93,7 +93,7 @@
 """
 
 SCORERS = {
-    'current_set_up': ['face_recognition'],
+    'current_set_up': ['vggface'],
```

Comment:
keep face_recognition?

Reply:
it is not working yet in this branch since Andrea's pull request is not resolved yet. But for the future, yes definitely and also with facevacs added